src/cdk/datetime/datepicker-input.ts

```diff
@@ -174,10 +174,10 @@ export class CdkDatepickerInput<D> implements AfterContentInit, ControlValueAcce
   private _disabled: boolean;
 
   /** Emits when a `change` event is fired on this `<input>`. */
-  @Output() readonly dateChange: EventEmitter<DatepickerInputEvent<D>>;
+  @Output() readonly dateChange = new EventEmitter<DatepickerInputEvent<D>>();
 
   /** Emits when an `input` event is fired on this `<input>`. */
-  @Output() readonly dateInput: EventEmitter<DatepickerInputEvent<D>>;
+  @Output() readonly dateInput = new EventEmitter<DatepickerInputEvent<D>>();
```

Comment:
These two outputs should be typed to `EventEmitter`, because the doc generator doesn't infer the type automatically.